pod/perlfunc.pod

```diff
@@ -4717,17 +4717,18 @@ The following blocks are more or less equivalent:
     open(my $fh, "-|") || exec "cat", "-n", $file;
     open(my $fh, "-|", "cat", "-n", $file);
 
-The last two examples in each block show the pipe as "list form", which is
-not yet supported on all platforms.  A good rule of thumb is that if
-your platform has a real L<C<fork>|/fork> (in other words, if your platform is
-Unix, including Linux and MacOS X), you can use the list form.  You would
-want to use the list form of the pipe so you can pass literal arguments
-to the command without risk of the shell interpreting any shell metacharacters
-in them.  However, this also bars you from opening pipes to commands
-that intentionally contain shell metacharacters, such as:
+The last two examples in each block show the pipe as "list form", which
+is not yet supported on all platforms.  A good rule of thumb is that if
+your platform has a real L<C<fork>|/fork> (e.g. your platform is Unix,
+including Linux and macOS, or you're using Perl 5.22 or later with
```

Comment:
This kind of implies that Windows has real fork if you use Perl 5.22 or later, which isn't the case; perhaps put this case outside the parenthetical.